Makefile

```diff
@@ -8,19 +8,23 @@ MAJOR := $(word 1,$(subst ., ,$(VERSION)))
 MINOR := $(word 2,$(subst ., ,$(VERSION)))
 MAJOR_MINOR_PATCH := $(word 1,$(subst -, ,$(VERSION)))
 
-all: hub chrome firefox chrome_debug firefox_debug standalone_chrome standalone_firefox standalone_chrome_debug standalone_firefox_debug
+all: hub chrome firefox opera chrome_debug firefox_debug opera_debug standalone_chrome standalone_firefox generate_standalone_opera standalone_chrome_debug standalone_firefox_debug generate_standalone_opera_debug
```

Comment:
Seems the standalone and standalone debug images are not being generated, can you please double check the Makefile?

Reply:
Done.